### PR TITLE
Update blogs.yml

### PR DESCRIPTION
**Pull request by Anushka-Kadam**
Thank you for your pull request! Please read and understand everything below

Please acknowledge that you have read and understood each of the following before submitting your pull request.

# _Do not_ delete any text other than where you are instructed.

**Maintainers: If any of these are left unchecked, do not merge.**

**Students: If one of them is not applicable to you -- Please check it anyways. DO NOT REMOVE ANYTHING!**

**You may check them after opening your pull request.**

You may do so by changing each `[ ]` to `[x]` Please take note of the whitespace as it matters.

- [x] Read and understood (see CONTRIBUTING.md)
- [x] Included a live link or preview screenshot
- [x] Images are `240 x 240` [w x h]
- [x] Resolved merge conflicts
- [x] Included a description of my change below

# Things done in this Pull Request

- I have added my blog post including my interview of another student's experience with the code-in.

student interview

preview

### CODE REVIEW

**Review by anshumanv — CHANGES_REQUESTED** [review]
Please upload the image.

**Comment by Anushka-Kadam** [comment]
I uploaded the image.

**Comment by ParthS007** [comment]
@Anushka-Kadam  you have to add the image in `gci17.fossasia.org/img/blog/` and Image should be of dimension `240*240` only.
`

File: _data/blogs.yml [hunk 1]
```diff
@@ -137,3 +137,8 @@
   title: Interview with a FOSSASIA Contributer | Ploypiti Piyaprapan
   img: aditya-blog-2.jpg
   author_name: Aditya Dutt
+
+- url: https://anushka264230468.wordpress.com/2017/12/16/first-blog-post/
+  title: A Student’s Experience
+  img: Student Interview Image.jpg
```
Comment (ParthS007) [inline _data/blogs.yml]: Don't leave space or use `Student-Interview-Image.jpg` or you can use small name also.

**Review by ParthS007 — CHANGES_REQUESTED** [review]

**Comment by Anushka-Kadam** [comment]
How does one do that because I can't get the image to appear when I preview it?

**Comment by ParthS007** [comment]
@Anushka-Kadam First add the image in the specified folder then push it.
Then please go through this https://pages.github.com/ and generate a preview link 👍

**Review by gabru-md — CHANGES_REQUESTED** [review]
Please Add your Image that you want to display.
And provide a preview link so that this can be done with  :)

**Comment by Anushka-Kadam** [comment]
I can't add the image. Every time I try, nothing shows up.

**Comment by ParthS007** [comment]
@Anushka-Kadam  Can you share the screenshot what error are you facing or can you explain what you are trying to do ?

**Comment by Anushka-Kadam** [comment]
I am trying to add a picture.
I've tried doing  method but when I click preview, it gives me the exact same text that I wrote up. It doesn't show me the image. I'm not sure if that's what it's supposed to do or if it is actually supposed to show me the picture because currently, I can't tell if it's working or not.

**Review by vaibhavsingh97 — CHANGES_REQUESTED** [review]
Can you please share deployment link

**Comment by Anushka-Kadam** [comment]
Sorry, what's a deployment link?

**Comment by vaibhavsingh97** [comment]
@Anushka-Kadam Are you using github UI to upload picture? Can you please join https://gitter.im/fossasia/fossasia for further discussion. Ping me

**Pull request closed without merging**